search-sentiments-map/app.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const PORT = process.env.PORT || 3000;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+// Create server that displays the webpage with the html, css, javascript, and png files in the public folder.
```

Comment:
I think this may need to be line wrapped.

Reply:
got it

Reply:
Fyi, Haylee said that the google standard line wrap is 100 chars, not 80 chars

Reply:
For JS it is 80 chars, for Java it is 100. 

https://google.github.io/styleguide/jsguide.html#formatting-column-limit

https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

Reply:
oh gotcha